test/controllers/CyaCategorisationControllerSpec.scala

```diff
@@ -314,7 +442,62 @@ class CyaCategorisationControllerSpec extends SpecBase with SummaryListFluency w
                 recordId = testRecordId,
                 category = 1,
                 categoryAssessmentsWithExemptions = 0,
-                measurementUnit = Some("Weight, in kilograms")
+                measurementUnit = Some("Weight, in kilograms"),
+                comcode = None
+              )
+
+              status(result) mustEqual SEE_OTHER
+              redirectLocation(result).value mustEqual routes.CategorisationResultController
+                .onPageLoad(testRecordId, Category1)
+                .url
+              //TODO update with new expected - Why was this TODO in main?
```

Comment:
was TODO in main from when we were rushing stuff on Friday. 

Looking at it now it should just need to go from updateGoodsRecord to updateCategoryForGoodsRecord.